vm-presets.md

```diff
@@ -0,0 +1,237 @@
+Virtual Machine Presets
+=============================
+
+`VirtualMachinePresets` are an extension to general `VirtualMachine`
+Configuration based heavily on `PodPresets` from Kubernetes. When a
```

Comment:
lower case c for configuration?
Also s/based/behaving/? Base could imply that they are technically based on the same src.